src/acvp.h

```diff
@@ -2539,7 +2581,7 @@ ACVP_RESULT acvp_set_2fa_callback (ACVP_CTX *ctx, ACVP_RESULT (*totp_cb) (char *
 
 ACVP_RESULT acvp_bin_to_hexstr (const unsigned char *src, unsigned int src_len, unsigned char *dest);
```

Comment:
Would also be nice if you change this declaration to char * for dest so no casts needed any longer!